docs/getting-started/development.md

```diff
@@ -0,0 +1,33 @@
+# Setup a development environment
+
+To add features to the repository follow the following procedure to setup a working development environment.
+
+## Installation (Development)
+Install the project and its dependancies to develop on the code.
+
+### Step 1 - install flit:
+```console
+pip install flit
+```
+
+### Step 2 - install the development code:
+```console
+flit install --deps develop --symlink
```

Comment:
This is almost never what you want. It installs all packages outside the active venv.

a `--python path/to/venv/bin/python` is needed

Reply:
@acidjunk Did you install flit in your user env or in the specific venv? I did not encounter this issue. I got it in the specific venv

Reply:
Ill add that to the docs, that flit needs to be installed in a env not globally.

Reply:
I installed it in an env: but it installed the packages outside the env (conda base python)

The whole point of setting up a symlinked dev version is to have 2 editable projects (IMHO)

Reply:
@hmvp encountered the same problem, still not sure when it happens exactly. The readme has tested instructions now.

Reply:
Added a warning in the documentation to explain what happens here.